### ISSUE TITLE
Convert script into a Class

### ISSUE THREAD

**Issue by DrkSephy** [DrkSephy/genetic-algorithms#4]
Currently, the source code is simply one big script. For accessibility of the genetic algorithm parameters (and code clarity), it should be converted into a Class.

**Comment by DrkSephy** [comment]
Implemented by c1e66db9d307d5d842040a889002f4b238fa5644.
